build.gradle

```diff
@@ -82,7 +82,7 @@ configure(subprojects) {
 
     dependencies {
         testCompile "junit:junit:$junitVersion"
-        compile 'woodser:monero-wallet-java:0.0.2-SNAPSHOT'
+        compile 'woodser:monero-wallet-java:0.1.1-SNAPSHOT'
```

Comment:
Probably rename to woodser:monero-java:0.1.1-SNAPSHOT

Reply:
Instead of adding the new jar dependency, it would be better to simply add this code to Bisq so it can be reviewed together with this PR.

Reply:
@wiz 

I actually restrained myself from doing that. I am exploring what I had earlier missed: the use of the **`apply plugin: 'maven' `** directive.